CHANGES.md

```diff
@@ -28,6 +28,18 @@ To be released.
        from `IValue` to `IValue?`.
      - Changed `extra` parameter type of `AppProtocolVersion.Sign` method
        from `IValue` to `IValue?`.
+ -  (Libplanet.Net) Changed swarm-related types due to removal of
+    'nullable keyword'. [[#3670]]
+     - Changed `options` parameter type of `Swarm` class constructor from
+       `SwarmOptions` to `SwarmOptions?`.
+     - Changed `consensusTransport` parameter type of `Swarm` class constructor
+       from `ITransport` to `ITransport?`.
+     - Changed `Swarm.Validators` property type
+       from `IReadOnlyList<BoundPeer>` to `IReadOnlyList<BoundPeer>?`.
+     - Changed `progress` parameter type of `Swarm.PreloadAsync` method
+       from `IProgress<BlockSyncState>` to `IProgress<BlockSyncState>?`.
+     - Changed return type of `Swarm.FindSpecificPeerAsync` method
+       from `Task<BoundPeer>` to `Task<BoundPeer?>`.
```

Comment:
Changes are not related to network-protocol. Can you move theses to `Backward-incompatible API changes`?